core/src/main/resources/db/cgds.sql

```diff
@@ -727,3 +729,10 @@ CREATE TABLE `info` (
 );
 -- THIS MUST BE KEPT IN SYNC WITH db.version PROPERTY IN pom.xml
 INSERT INTO info VALUES ('1.1.0');
+
+--1.2.0 specific changes:
```

Comment:
Ok, I updated the `cgds.sql`. Actually I had apparently already done that before, so I just needed to remove these lines and update the 1.1.0 to 1.2.0.